command.php

```diff
@@ -348,6 +350,58 @@ public function is_woocommerce_blocks_active(): bool {
 
 		return 0 === $result->return_code;
 	}
+
+	/**
+	 * Sets up the WooCommerce Stripe gateway.
+	 */
+	public function setupStripeGateway() {
```

Comment:
Similar to my previous comment, I wonder if we can make this function `private` as well to prevent that it shows up in the help menu:

<img width="764" alt="Screenshot 2022-07-08 at 12 08 57" src="https://user-images.githubusercontent.com/3323310/177971363-cda466b9-4aa4-4860-9fe3-991435cbf3ce.png">

In addition, what are your thoughts about putting the Stripe payment behind an additional parameter, e.g. `--stripe`? That way, this part will only be called if someone needs to run tests using Stripe.

Reply:
Yep! Makes sense, on it! 🙌